content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -158,6 +158,7 @@ different Kubernetes components.
 | `RotateKubeletServerCertificate` | `false` | Alpha | 1.7 | 1.11 |
 | `RotateKubeletServerCertificate` | `true` | Beta | 1.12 | |
 | `RunAsGroup` | `true` | Beta | 1.14 | |
+| `SeccompDefault` | `false` | Alpha | 1.22 | 1.23 |
```

Comment:
Because v1.23 is a future version (?), you can leave the last column empty.

Reply:
Yep, done :+1: